### ISSUE TITLE
Option to have a persistent notification

### ISSUE THREAD

**Issue by daniel-sousa-me** [xbmc/Kore#996]
## Feature suggestion

A lot of manufacturers modify their versions of Android to be too eager to kill background services.
Whenever I start playing media through other remote, Kore doesn't know it because it's usually not running.

I'd like to be able to have a persistent notification, like many other apps do, to tell Android not to kill the service.

### Context
Lots of times I start playing media on my TV through my laptop because I'm finishing up something. But then I want to be able to easily control Kodi on my phone.

I know that opening Kore manually doesn't sound like too much work, but with this, I don't even have to unlock my phone.
Also, when some media is playing, I have basic controls on my smartwatch. But that doesn't work if Kore isn't running.

**Comment by gusandrianos** [comment]
Hi @daniel-sousa-me! A persistent notification isn't immune to killing from aggressive background process management. Would a quick-setting toggle be as helpful? 

Have you tried changing the energy settings of Kodi from the system settings? Idk what version of Android you're using but it's likely you'll have an unrestricted option.

**Comment by daniel-sousa-me** [comment]
Yes, I have tried fiddling with every possible setting. Before this one, I had a Samsung phone and its killing just drove me mad. I now have one that has few modifications, but it still is more aggressive than I'd like :/

I have persistent notifications for Telegram FOSS, Home Assistant, KDE Connect, and K-9 Mail (also NFC for some reason. This one I don't understand). You are right that they're not immune, but missing notifications is the exception in all these. Otoh Kore seldom catches up if I haven't been watching recently and sometimes it even stops working if I pause for a bit.

I think K-9 is the only one that gets the notifications consistently. I may try to find out if they're using any other tricks (notifications certainly aren't coming through Google stuff, because I run the only mail server that I use)
